x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameUsageIT.java

```diff
@@ -105,8 +105,8 @@ public void testUsage() throws Exception {
         usageAsMap = entityAsMap(usageResponse);
         // we should see some stats
         assertEquals(3, XContentMapValues.extractValue("data_frame.transforms._all", usageAsMap));
-        assertEquals(1, XContentMapValues.extractValue("data_frame.transforms.started", usageAsMap));
-        assertEquals(2, XContentMapValues.extractValue("data_frame.transforms.stopped", usageAsMap));
+        // TODO: due to auto-stop we only see stopped data frames
+        assertEquals(3, XContentMapValues.extractValue("data_frame.transforms.stopped", usageAsMap));
```

Comment:
It seems to me that there may be a runtime problem where the single task that was started has not actually ticked over to `STOPPED` by the time this method is called, maybe an `assertBusy`?

Reply:
`startAndWaitForTransform` called on line 70 does the `assertBusy`, see changes in `DataFrameRestTestCase`